networklibrary/src/main/java/com/eis/communication/network/SMSNetDictionary.java

```diff
@@ -0,0 +1,60 @@
+package com.eis.communication.network;
```

Comment:
I'd move every implementation with SMS in a different package. We worked hard to be as generic as possible with the interfaces in `communication.network`, let's not waste what we've done.

Reply:
My problem is I really struggle to work with generalization, I tend to work on a concrete example and THEN generalize.